standalone/src/main/java/com/criteo/publisher/samples/standalone/NativeRecyclerViewActivity.java

```diff
@@ -0,0 +1,168 @@
+package com.criteo.publisher.samples.standalone;
+
+import static com.criteo.publisher.samples.standalone.CriteoSampleApplication.CRITEO_NATIVE_AD_UNIT;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+import android.widget.ImageView;
+import android.widget.TextView;
+import androidx.annotation.NonNull;
+import androidx.annotation.Nullable;
+import androidx.appcompat.app.AppCompatActivity;
+import android.os.Bundle;
+import androidx.recyclerview.widget.LinearLayoutManager;
+import androidx.recyclerview.widget.RecyclerView;
+import androidx.recyclerview.widget.RecyclerView.ViewHolder;
+import com.criteo.publisher.CriteoErrorCode;
+import com.criteo.publisher.advancednative.CriteoMediaView;
+import com.criteo.publisher.advancednative.CriteoNativeAd;
+import com.criteo.publisher.advancednative.CriteoNativeAdListener;
+import com.criteo.publisher.advancednative.CriteoNativeLoader;
+import com.criteo.publisher.advancednative.CriteoNativeRenderer;
+import com.criteo.publisher.advancednative.RendererHelper;
+import com.squareup.picasso.Picasso;
+import java.util.ArrayList;
+import java.util.List;
+
+public class NativeRecyclerViewActivity extends AppCompatActivity {
+
+  private MyAdapter adapter;
+  private CriteoNativeLoader nativeLoader;
+
+  private int dummyContentCount = 12;
```

Comment:
Both  `dummyContentCount` and `nativeAdPosition` should be constants:
```
private static final int DUMMY_CONTENT_COUNT = 12;
private static final int NATIVE_AD_POSITION = 3;
```

Reply:
Done.